docs/section19/19-7.md

```diff
@@ -0,0 +1,87 @@
+    - Case 3: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and a zone served by the HVAC in the baseline has an air distribution effectivenss of greater than 1 in the proposed (Exception 2) then the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and zone_air_distribution_effectiveness_greater_than_1 == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 2 may be applicable. A manual check for this exception is recommended otherwise fail."`  
+    - Case 4: Else if the modeled baseline minimum OA cfm are greater than the modeled proposed minimum OA cfm and the hvac system only serves lab spaces (Exception 4) the outcome is UNDETERMINED: `elif modeled_baseline_total_zone_min_OA_CFM > modeled_proposed_total_zone_min_OA_CFM and hvac_system_serves_only_labs == true: outome = "UNDETERMINED" and raise_message "For <insert hvac.id> the modeled baseline minimum ventilation system outdoor air intake flow CFM appears higher than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. It appears as though G3.1.2.5 Exception 4 may be applicable. This exception states that for baseline systems serving only laboratory spaces that are prohibited from recirculating return air by code or accreditation standards, the baseline system shall be modeled as 100% outdoor air. A manual check for this exception is recommended otherwise fail."`  
+    - Case 5: Else if the modeled baseline minimum OA cfm are less than the modeled proposed minimum OA cfm the outcome is FAIL: `elif modeled_baseline_total_zone_min_OA_CFM < modeled_proposed_total_zone_min_OA_CFM: outome = "FAIL" and raise_message "For <insert hvac.id> the modeled minimum ventilation system outdoor air intake flow CFM appears lower than the minimum ventilation system outdoor air intake flow CFM modeled in the proposed design. Fail unless G3.1.2.5 Exception 3 is applicable. This exception states that where the minimum outdoor air intake flow in the proposed design is provided in excess of the amount required by the building code or the rating authority, the baseline building design shall be modeled to reflect the greater of that required by either the rating authority or the building code and will be less than the proposed design. "`  
```

Comment:
If we decide to differentiate mistakes that are likely to lead to more conservative outcomes, this one may be a candidate.